src/lib.rs

```diff
@@ -356,7 +356,7 @@ pub fn get_mime_type_opt(search_ext: &str) -> Option<Mime> {
 /// Returns `None` if `search_ext` is empty or an associated extension was not found.
 #[deprecated(
     since = "2.0.0",
-    note = "use `from_ext(search_ext).first_as_str()` instead"
+    note = "use `from_ext(search_ext).first_raw()` instead"
 )]
 pub fn get_mime_type_str(search_ext: &str) -> Option<&'static str> {
     from_path(search_ext).first_raw()
```

Comment:
Oops, looks like I made a mistake here which I missed because I removed the tests for the deprecated functions. This should use `from_ext()` like the deprecation message, not `from_path()`.

Reply:
oh really, I didn't notice mistake at first, will correct